documentatie/flowcharts/data-entry-state.md

```diff
@@ -0,0 +1,23 @@
+# Data entry state
+
+This document describes the states a data entry can have.
+The transition labels describe the endpoint that is used for performing the transition.
+The "save" endpoint, used to for [First/Second]EntryInProgress states is kept out, because Mermaid doesn't render transitions to itself too well.
```

Comment:
```suggestion
The "save" endpoint, used to for [First/Second]EntryInProgress states is kept out, because Mermaid doesn't render self-loops too well.
```